include/string_utils.h

```diff
@@ -224,6 +224,7 @@ inline bool is_empty(const char *str) noexcept
 
 // case-insensitive comparisons
 bool ciequals(const char a, const char b);
+[[nodiscard]] bool ci_str_equals(std::string_view a, std::string_view b);
```

Comment:
Does `iequals(a, b)` not handle two string_views?

Reply:
+1